src/AssimpWorker/worker/Worker.cpp

```diff
@@ -73,7 +73,8 @@ namespace AssimpWorker {
 	void Worker::importSingleColladaFile() {
 		std::istream& response = storageService.retrieveFile(currentWorkUnit->sourcePath);
 		// Patches welcome...
-		Poco::TemporaryFile tmp;
+		std::string decompressionPath = Configuration::getInstance().get("decompression-path").as<std::string>();
```

Comment:
On second thought, fixing this temporary file business sounds like another feature branch entirely. I'll note it down for the future and let it slide here, since this line is consistent with the handling of teporaries elsewhere in `Worker.cpp`.